docs/dcp2_system_design.rst

```diff
@@ -960,221 +960,91 @@ Project-level matrices
-                      {
-                        "name": "file_1.loom",
-                        "url": "https://.../fetch/repository/files?...",
-                      }
-                    ]
-                  }
-                }
-              }
-            }
-          }
-        },
-        "Mus musculus": {
-          "developmentStage": {
-            "adolescent": {
-              "organ": {
-                "pancreas": {
-                  "libraryConstructionApproach": {
-                    "10X v2 sequencing": [
-                      {
-                        "name": "file_2.loom"
-                        "url": "https://.../fetch/repository/files?...",
-                      }
-                    ],
-                    "Smart-seq2": [
-                      {
-                        "name": "file_2.loom",
-                        "url": "https://.../fetch/repository/files?...",
-                      }
-                    ]
-                  }
-                }
-              }
-            }
-          }
-        }
-      }
+        "text": "Contributor-generated cell type identifier files",
+        "ontology": "data:2655",
+        "ontology_label": "Cell type identifier"
     }
 
-for a project with at least one stratified matrix file or::
+``Subgraph Changes for CGMs (managed access)``
+
+For managed access datasets, raw data and metadata may not be available. Ideally we would link CGMs to the specific sequence files which have created them. However, if the sequence file is not present in the metadata (due to controlled access or the information not being known), the analysis_file would be linked to a biomaterial in the metadata schema.
+
+There is currently one graph shape being tested by Azul and other components as a candidate subgraph, in which cell suspension links directly to analysis file.
+Specifically:
+Donor —> Specimen —> Cell Suspension —> Analysis File
+
+
+.. figure:: media/managed-cgms.png
+   :align: center
+
+
+
+It is important to note here that this means we have not tested, and hence pipelines and Azul may not be able to process or index, subgraphs in which biomaterials other than ‘cell_suspension’ link to the analysis file.
+
+At the moment, no post-processing is performed on contributor-generated matrices. This will be kept the same. Users will be able to download them in exactly the same file format as they were provided by the contributor. For example, if the contributor provided a ZIP archive of multiple matrix files, the user will need to download that archive and extract it locally.
+
+``Stratification``
 
-    [
-        {
-           "url": "https://.../fetch/repository/files?...",
-           "name": "file_1.loom"
-        }
-    ]
+Stratification can be produced by linking the CGM analysis file to specific sequence_file or biomaterial entities. If there are matrices which are generated from a subset of sequence files, we are able to link only that subset of sequence files to the CGM.
```

Comment:
```suggestion
Stratification can be produced by linking the CGM analysis file to specific ``sequence_file`` or biomaterial entities. If there are matrices which are generated from a subset of sequence files, we are able to link only that subset of sequence files to the CGM.
```

Reply:
This is the intended use of double back ticks. Please apply it consistently across your text.